star-ros2/src/star_spi_bridge/include/star_spi_bridge/spi_message_converter.hpp

```
@@ -11,36 +11,30 @@
 #include "star/v1/motor_control.pb.h"
 #include "star/v1/telemetry.pb.h"
 
-namespace star_spi_bridge
-{
+namespace star_spi_bridge {
 
 class SpiMessageConverter {
 public:
-  struct Parameters
-  {
+  struct Parameters {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: wheel_base, wheel_radius, ticks_per_rev

(cppcoreguidelines-pro-type-member-init)